src/olympia/migrations/879-remove-t-shirt-order-field.sql

```diff
@@ -0,0 +1,2 @@
+ALTER table `users` 
+    DROP column `t_shirt_requested`;
```

Comment:
can you add `DELETE FROM waffle_switch WHERE name = 't-shirt-orders';` to remove the waffle switch too?